raiden_contracts/contract_manager.py

```diff
@@ -74,10 +81,21 @@ def __init__(self, path: Union[Path, Dict[str, Path]]) -> None:
                     self.contracts = precompiled_content['contracts']
                     self.overall_checksum = precompiled_content['overall_checksum']
                     self.contracts_checksums = precompiled_content['contracts_checksums']
-                    self.contracts_version = precompiled_content['contracts_version']
+                    try:
+                        self.plain_version = PlainVersion(
+                            precompiled_content['plain_version'],
+                        )
+                    except KeyError:  # Assuming an older format
+                        self.plain_version = PlainVersion(
+                            precompiled_content['contracts_version'],
+                        )
+                    self.contracts_version = FlavoredVersion(
+                        precompiled_content['contracts_version'],
```

Comment:
So the contracts version is by default the flavored version now?

Reply:
Yes.